IM_calculation/IM/Burks_Baker_2013_elastic_inelastic.py

```diff
@@ -41,8 +41,19 @@
 
 
 def Bilinear_Newmark_withTH(
-    period: np.ndarray, z: float, dy: float, alpha: float, ag: np.ndarray, dt: float
+    period: np.ndarray,
+    z: float,
+    dy: float,
+    alpha: float,
+    ag: np.ndarray,
+    Dtg: float,
+    Dt: float,
```

Comment:
Should use lower case dt dtg, to be consistent with the rest of the codebase.
Consistency with ported source code is secondary (I think)

Reply:
using dt, instead of Dt
pros: consistent with other steps we have
cons: if someone tried to search for variables after reading the doc for imported (and have no experience in our workflow) will have a hard time.

i would personally chose it to match the doc for ported source, because the most likely scenario for us to modify the code is when we have doubt and need to match it with original to find out the cause.

Reply:
Changed Dt -> dt 
There are bunch of other variables Ki, Ddi, Dpi etc keeping the same MATLABic style - but lets leave them out.